### PR TITLE
Removed unused libraries, and added logic for missing ion parameter file in topsStack

### PR DESCRIPTION
**Pull request by bjmarfito**
- Removed unused and duplicate library imports in `mdx.py` and `unwrapComponents.py`
- Added check if the ion parameter file actually exists for ionospheric correction in `stackSentinel.py`

### CODE REVIEW

File: contrib/stack/topsStack/coregSwathSLCProduct.py [hunk 1]
```diff
@@ -1,3 +1,5 @@
+#!/usr/bin/env python3
+
```
[the remark this answers is not in the document]
Comment (bjmarfito) [reply contrib/stack/topsStack/coregSwathSLCProduct.py:2]: I see. I will remove this part. Thank you!

[the remark this answers is not in the document]
Comment (bjmarfito) [reply contrib/stack/topsStack/coregSwathSLCProduct.py]: Done. Thank you @rtburns-jpl for checking it out.

File: contrib/stack/topsStack/stackSentinel.py [hunk 2]
```diff
@@ -1010,10 +1010,12 @@ def main(iargs=None):
 
 
     #do ionosphere estimation
-    if inps.param_ion is not None:
+    if inps.param_ion is not None and os.path.isfile(inps.param_ion) == True:
```
Comment (rtburns-jpl) [inline contrib/stack/topsStack/stackSentinel.py:1013]: I think the conditions here can be a bit simpler like this, and the messages would be more helpful:
```py
if inps.param_ion is None:
    print("Ion parameter file is not specified. Ionospheric estimation will not be done.")
elif not os.path.isfile(inps.param_ion):
    print("Ion parameter file is missing. Ionospheric estimation will not be done.")
else:
    <do iono>
```

Comment (bjmarfito) [reply contrib/stack/topsStack/stackSentinel.py]: Done. Thank you @rtburns-jpl for your suggestions.

**Review by rtburns-jpl — APPROVED** [review]
LGTM, thanks!

**Pull request merged**